### PR TITLE
Add a modal confirmation before installing WaveShell

### PR DESCRIPTION
**Pull request by adred**

### CODE REVIEW

File: src/app/common/common.tsx [hunk 1]
```diff
@@ -99,23 +99,57 @@ class Toggle extends React.Component<{ checked: boolean; onChange: (value: boole
...
 > {
+    generatedId;
+    static idCounter = 0;
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            checkedInternal: this.props.checked !== undefined ? this.props.checked : Boolean(this.props.defaultChecked),
+        };
+        this.generatedId = `checkbox-${Checkbox.idCounter++}`;
```
Comment (sawka) [inline src/app/common/common.tsx:120]: do you need this id?  it doesn't seem to be used

**Comment by sawka** [comment]
I don't think the DB migration is needed since a nil map[string]bool will work just fine for read-only access in go and you're doing the proper nil check in cmdrunner when setting a new flag.

File: src/app/common/common.tsx (hunk 1 above)
Comment (adred) [reply src/app/common/common.tsx:120]: Yes. In case multiple instances of Checkbox are rendered. It's used as `htmlFor` value.

**Comment by adred** [comment]
> I don't think the DB migration is needed since a nil map[string]bool will work just fine for read-only access in go and you're doing the proper nil check in cmdrunner when setting a new flag.

I think we do? `confirmflags.showShellPrompt` has to be set to true by default otherwise the prompt won't show at all.

**Pull request merged**